index.ts

```diff
@@ -50,85 +51,140 @@ export const setupPlugin: BigQueryPlugin['setupPlugin'] = async (meta) => {
     )
 
     const credentials = JSON.parse(attachments.googleCloudKeyJson.contents.toString())
-    global.bigQueryClient = new BigQuery({
-        projectId: credentials['project_id'],
-        credentials,
-        autoRetry: false,
-    })
 
-    global.bigQueryTable = global.bigQueryClient.dataset(config.datasetId).table(config.tableId)
+    try {
+        global.bigQueryClient = new BigQuery({
```

Comment:
does their constructor do any calls 🤷‍♀️

Reply:
Yeah not sure tbh, I think not but… we can refine later if neede